README.md

```diff
@@ -1 +1,170 @@
+<p align="right"><a href="#what-is-it"><sup>▴ Back to top</sup></a></p>
+Install the spello via `pip`
+
+```bash  
+$ pip install spello
+```  
+
+<h2 align="center">⚡ ️Getting Started</h2> 
+<p align="right"><a href="#what-is-it"><sup>▴ Back to top</sup></a></p>
+  
+#### 1. **Model Initialisation**
+Initialise the model for one of the suppored languages. 
+```python  
+>>> from spello.model import SpellCorrectionModel  
+>>> sp = SpellCorrectionModel(language='en')  
+```  
+
+#### 2. Model Training
+Here, you can provide two types of training data to the model
+- List of text or sentences.
+- Dict having word and their corresponding count.
+
+**Training providing list of sentences**
+```python 
+>>> sp.train(['my name is aman', 'this is a text corpus'])
+```
+**Training providing words counter**
+```python 
+>>> sp.train({'my': 2, 'name': 1, 'aman': 1, 'text': 10, 'corpus': 5})
+```
+> List of text is a recommended type for training data as here model also tries to learn context in which words are appearing, which further help to find best possible suggestion in case more than one suggestions are suggested by symspell or phoneme model
+
+#### 3. Model Prediction
+```python  
+>>> sp.spell_correct('my naem is naman')  
+{'original_text': 'my naem is naman',
+ 'spell_corrected_text': 'my name is aman',
+ 'correction_dict': {'naem': 'name', 'naman': 'aman'}
+}
+```  
+
+#### 4. Save Model
+Call the save method to save the trained model at given model dir 
+```python  
+>>> sp.save(model_save_dir='/home/ubuntu/')
+'/home/ubuntu/model.pkl' # saved model path
+```  
+
+#### 5. Load Model 
+Load the trained model from saved path, First initialise the model and call the load method
+```python  
+>>> from spello.model import SpellCorrectionModel
+>>> sp = SpellCorrectionModel(language='en')
+>>> sp.load('/home/ubuntu/model.pkl')
+```  
+
+#### 6. Customize Configuration of Model
+Here, you are also provided to customize various configuration of the model like 
+1. Setting minumum and maximum length eligible for spellcorrection
+```python  
+>>> sp.config.min_length_for_spellcorrection = 2 # default is 3
+>>> sp.config.max_length_for_spellcorrection = 20 # default is 15
+```  
+2. Setting Max edit distance allowed for each char level for symspell and phoneme model
+```python
+>>> sp.config.symspell_allowed_distance_map = {2:0, 3: 1, 4: 2, 5: 3, 6: 3, 7: 4, 8: 4, 9:5, 10:5, 11:5, 12:5, 13: 6, 14: 6, 15: 6, 16: 6, 17: 6, 18: 6, 19: 6, 20: 6}
+# above dict signifies max edit distance possible for word of length 6 is 3, for length 7 is 4 and so on..
+```
+*To reset to default config*
+```python
+>>> sp.set_default_config()
+```
+there are many more configurations which you can set, check this <a href="https://github.com/hellohaptik/spello/blob/master/spello/config.py">file</a> for more details
+
+
+## Get Started with Pre-trained Models
+Here are few simple models. They are trained on 30K news + 30k wikipedia sentences.
```

Comment:
"We have trained a basic model on on 30K news + 30k wikipedia sentences". Avoid starting sentences form "Here..."

Reply:
Done